Session.cc

```diff
@@ -52,7 +55,10 @@ Session::~Session() {
     }
     frames.clear();
     outgoing->close();
-    
+    if( _ani_obj ) {
```

Comment:
Can we make _ani_obj a unique_ptr to avoid this?

Reply:
I have done that.